robottelo/ui/contentviews.py

```diff
@@ -450,6 +465,37 @@ def add_remove_errata_to_filter(self, cv_name, filter_name,
         else:
             self.click(locators['contentviews.remove_errata'])
 
+    def edit_erratum_date_range_filter(
+            self, cv_name, filter_name, errata_types=None, date_type='updated',
+            start_date=None, end_date=None, open_filter=True):
+        """Edit Erratum Date Range Filter"""
+        allowed_errata_types = ['security', 'enhancement', 'bugfix']
+        allowed_date_types = ['updated', 'issued']
+        if not errata_types:
+            # one errata type is mandatory
+            errata_types = ['security']
+        if date_type not in allowed_date_types:
+            raise UIError('date type "{0}" does not exit'.format(date_type))
+        if open_filter:
+            self.go_to_filter_page(cv_name, filter_name)
+        for errata_type in allowed_errata_types:
+            errata_type_element = self.wait_until_element(
+                locators['contentviews.erratum_type_checkbox'] % errata_type)
+            if errata_type_element:
+                checked = errata_type_element.is_selected()
+                if ((errata_type in errata_types and not checked) or
+                        (errata_type not in errata_types and checked)):
+                    # click on it only if it's checked and not in the selection
+                    # or not checked and in the selection
+                    errata_type_element.click()
+            else:
+                raise UIError(
+                    'Cannot find the check box element %s .', errata_type)
+        self.click(locators['contentviews.erratum_date_type'] % date_type)
+        self.set_calendar_date_value('start_date', start_date)
```

Comment:
set_calendar_date_value will be changed (see above comment) 
and this function will be changed to not update the value with None values